packages/node_modules/samples/browser-plugin-meetings/app.js

```diff
@@ -502,7 +502,19 @@ document.getElementById('updateVideo').addEventListener('click', () => {
   });
 
   if (media.sendVideo) {
-    meeting.getMediaStreams({sendVideo: true}, {video: videoSelect.value ? {deviceId: {exact: videoSelect.value}} : media.sendVideo})
+    // Sending a boolean to `getMediaStreams` lets the browser decide which device to use.
```

Comment:
Making this device selector code match the one from `updateAudio`. This way is more verbose and easier to follow from a sample code.